frontend/src/store/courses-management/actions.ts

```diff
@@ -31,19 +28,15 @@ const getCourses = createAsyncThunk<
 });
 
 const updateCategory = createAsyncThunk<
```

Comment:
We should update swagger too

Reply:
@Liza-Veis I didn't change backend api, structure of the response is same

Reply:
Sorry, didn't notice it